packages/components/src/DateTimePickers/Date/Manager/Manager.component.js

```diff
@@ -11,32 +11,35 @@ import {
 } from '../date-extraction';
 
 function ContextualManager(props) {
-	// eslint-disable-next-line no-use-before-define
+	function getDateOptions() {
+		return {
+			dateFormat: props.dateFormat,
+			useUTC: props.useUTC,
+			required: props.required,
+			timezone: props.timezone,
+		};
+	}
```

Comment:
move this function definition to get rid of eslint "no-use-before-define" warning